schedule/security/sle16/agama_remote_s390x.yaml

```diff
@@ -4,6 +4,7 @@ description: >
   Perform interactive installation with agama for s390x worker.
 schedule:
   - yam/agama/boot_agama
+  - yam/agama/agama_arrange
```

Comment:
there is not need to add these two files back, they were just for visualization.